lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -33,6 +33,8 @@
     __chakraLibrary.raiseFunctionArgument_NeedFunction = platform.raiseFunctionArgument_NeedFunction;
     __chakraLibrary.functionBind = platform.builtInJavascriptFunctionEntryBind;
     __chakraLibrary.objectDefineProperty = _objectDefineProperty;
+    __chakraLibrary.toString = platform.String;
```

Comment:
I'm not certain if this is the right way to access String()? (copied logic from intl.js)

Reply:
`String()` is actually not the same as the specification's ToString() operation; the former has a special case for Symbols.  ToString(symbol) will throw; `String(symbol)` will not.  Attempting to sort an array with any symbols in it using the default comparator should throw, and it does at least in V8:
```
> ([ Symbol(), Symbol(), Symbol() ]).sort()
TypeError: Cannot convert a Symbol value to a string
    at Array.sort (native)
```

I'm not sure there's a clean way to trigger ToString() coercion other than
```js
let stringifiedValue = "" + valueToStringify;
```